gapic/batch/common.yaml

```diff
@@ -0,0 +1,5 @@
+common:
+  batch_apis: '*'
+  batch_languages: '*'
+  api_config_pattern: ${THISDIR}/../api/artman_*.yaml
+  artman_language_yaml: ${THISDIR}/../lang/common.yaml
```

Comment:
I thought we only wanted to remove them from configs that are consumed by things running in tasks, e.g. toolkit, and not configs that are processed by artman while constructing pipelines. E.g. there are still relative paths in https://github.com/googleapis/googleapis/blob/master/gapic/lang/common.yaml
Should I change them as well?